tensorflow_model_optimization/python/core/common/keras/compression/internal/optimize.py

```diff
@@ -232,6 +232,7 @@ def _map_to_training_weights(
   # user has to pass in the information that 'kernel' is the first
   # variable, 'bias' is the second variable, and so on.
   # TODO(tfmot): see if Keras can introduce changes to simplify this.
+  original_weights = []
```

Comment:
Can remove both TODOs from line 239 - 253 given that you've determined that the order from 
_TrainingWrapper(layer).get_weights() is all the weights in the wrapper first and then the layer.

Reply:
Done.